semcore/input/src/style/input.shadow.css

```diff
@@ -78,6 +78,11 @@ SAddon {
   padding: 0 var(--intergalactic-spacing-2x, 8px);
   height: 100%;
   color: var(--intergalactic-icon-secondary-neutral, #a9abb6);
+
+  button [data-ui-name="ButtonLink.Addon"],
+  a [data-ui-name="Link.Addon"] {
+    margin-bottom: 0;
```

Comment:
@ilyabrower it's not button/a, it's a span that has that margin-bottom

Reply:
yes, but this span is inside button

Reply:
sorry, I don't know wtf happened, I just looked at the branch and apparently I haven't pulled your changes, I don't know what I was testing on Friday %)

Now I see that the bottom margin is gone, but unfortunately it now looks misaligned (too low) in Firefox. Is there a way to fix it?
<img width="316" alt="image" src="https://github.com/user-attachments/assets/a7e88ebd-adcd-4515-a7d3-a2cc29032583">

Reply:
Recheck it please. I changed display type and checked in that example and on ButtonLink page example